Python_sem_4/DZ2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите функцию принимающую на вход только ключевые параметры и возвращающую словарь, где ключ — значение
+# переданного аргумента, а значение — имя аргумента. Если ключ не хешируем, используйте его строковое представление.
+
+# *args
+# hash
+
+def my_func (**kwargs):
+    for i in kwargs:
```

Comment:
Если итерируемая переменная не используется, то пишут просто нижнее подчеркивание.
```suggestion
    for _ in kwargs:
```